client-runtime/src/main/java/com/microsoft/rest/v2/annotations/RequestContentType.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines the content type (MIME type) for the request body.
+ *
+ * If the Content-Type header present in {@link Headers} annotation then this annotation override it because this annotation is more specialized.
+ * If the Content-Type header is passed as parameter via {@link HeaderParam} annotation then {@link HeaderParam} takes precedence over this annotation,
+ * because that is what user selected during runtime.
+ * The value of this annotation overrides "Content-Type" header added in any interceptors, and "Content-Type" header added by the HTTP itself.
+ *
+ * The precedence is @Header("Content-Type") [dynamic] > @RequestContentType(value) [static] > @Headers("Content-Type: ..") [static]
```

Comment:
it should be `@HeaderParam("Content-Type")` not `@Header("Content-Type")`. We have `@Headers` (plural) which is applicable only in method level and `@HeaderParam` is applicable in parameter level.

`@Headers` used to define values statically. `@HeaderParam` is used to set values during runtime.